frontend/src/components/NewsFeed.jsx

```diff
@@ -1,15 +1,44 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
 import "./style/NewsFeed.scss";
 import Postcard from "./Postcard";
 import LiveContainer from "./LiveContainer";
+import PostField from "./PostField";
 
 function NewsFeed() {
+  const [posts, setPosts] = useState([]);
+  const [offsetPosts, setoffsetPosts] = useState(0);
+  // il faut ajouter une limite de recharge pour le nombre de post. Sinon l'appel axios se fait même s'il n'y a plus rien à afficher
```

Comment:
Do not leave comments about what to do later: either you do it, or you add this to the backlog